### DISCUSSION TITLE
Almost have the installation working, but get Authentication Failed

### DISCUSSION THREAD

**Discussion by jenny787** [outline/outline#5462] [Self Hosting]
After many attempts of this difficult installation, and many guides used, I have almost got Outline working.  I can post configs, but here is my summary.

Outline is installed using docker on a machine with ports 80,443 open to the puublic.
A domain is assigned to the machine, A name.  outline.domain2.com
Another domain on another machine runs Authentik, auth.domain1.com

I can access the outline login page.
Minio is configured for outline, and accessible separately using localhost.
I use OpenID button to login, which sends me to Authentik.
Authentik is successfull with the login, even the check user test works fine.
Application is authorized.  it redirects me back to outline.
Now, it says Authentication Failed, we were unable to sign you in.  please try again later.

I checked all my configs and everything looks correct.  Again, Authentik claims it logs in successfully.
The redirect URI is correct.  https://outline.domain2.com/auth/oidc.callback

In the env file, I have correctly configured the AWS block with the Minio settings, and the OIDC block.

Not sure what else I can do, thank you.

**Comment by tommoor** [comment]
It sounds like the outline server process is not able to access authentik, usually this is a DNS issue – the Outline container is not able to resolve the Authentik url.

**Reply by jenny787** [reply]
So this must be the container network issue I have read about on some of the localhost guides.  But what can I do?  The authentik server is fully accessible from outside with it's own domain and everything.  I also have hundreds of containers running in this fashion, all are able to see one another.
One suggestion I read was to add outline, minio, postgres, etc all to the hosts file.  But that was for a localhost only install, and this is a production install.  If I add an entry to the auth server in the hosts file, would that work?  Like:
192.168.1.22 auth.domain1.com
?
